CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ MESSAGE(STATUS "Support for nearest-neighbor-gradient mapping and tetrahedra: ${
 
 find_package(Boost 1.65.1 REQUIRED COMPONENTS log log_setup system program_options filesystem unit_test_framework)
 
-find_package(VTK REQUIRED)
+find_package(VTK REQUIRED COMPONENTS CommonCore CommonDataModel IOXML IOLegacy)
```

Comment:
Cool idea, I guess you need to adapt this in order to start the CI properly.

Reply:
Committed by mistake